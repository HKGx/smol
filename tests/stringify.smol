let bools := [true, false, true, false]

for b in bools: print(str(b))

let ints := [1, 2, 3, 4]

for i in ints: print(str(i))

let strings := ["one", "two", "three", "four"]

for s in strings: print(str(s))