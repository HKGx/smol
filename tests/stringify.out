True
False
True
False
1
2
3
4
one
two
three
four
None
